web/src/main/java/com/google/accompanist/web/WebView.kt

```diff
@@ -250,6 +250,10 @@ internal fun WebContent.withUrl(url: String) = when (this) {
  * See [Loading] and [Finished].
  */
 sealed class LoadingState {
+    /**
+     * Describes a WebView that has not yet loaded for the first time.
```

Comment:
Don't forget to update `Finished` comment

Reply:
Good spotting, thank you!